### PR TITLE
Removed old spiceinit makefiles and fixed test

### PR DESCRIPTION
**Pull request by jessemapel**
## Description

- Removed the old spiceinit Makefile tests. There is still one test that tests the shapemodel configuration. It is done via preferences file and it's not clear how to modify that in a test without contaminating other tests.
- Fixed spiceinit ck config test. The MRO update caused the FK to change which the test looks at. We could run the separate kernel dbs to get what should be there, but that's largely just duplicating the code in spiceinit. Until we figure out what we want to do about isolating the kernels used for testing spiceinit, I would recommend just manually updating this when MRO does a new FK release every couple years.

## Related Issue

No issue, tests on Jenkins are currently failing.

## Motivation and Context

This removes the duplicate tests so that we're not wasting time re-checking.

## How Has This Been Tested?

Tested on Ubuntu with the updated data area and the ck config test now passes. Old spiceinit tests are now properly ignored on Mac and Linux too.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Review by acpaquette — COMMENTED** [review]
Overall looks good, maybe we could remove the `_v##` from the fk comparison in the future to make this test a little less brittle to kernel file names? There is one last test that also needs to be fixed, `kerneldbgen_app_test_extra` is failing due to the kernel file change.

**Comment by jessemapel** [comment]
kerneldbgen needs updated truthdata as it's a Makefile test. I am out of time today to get to that. @scsides @kberryUSGS Can one of your update its test data on your support time?

File: isis/tests/FunctionalTestsSpiceinit.cpp [hunk 1]
```diff
@@ -188,7 +189,12 @@ TEST(Spiceinit, TestSpiceinitCkConfigFile) {
   EXPECT_PRED_FORMAT2(AssertQStringsEqual, instrumentPointing[0], "Table");
   EXPECT_PRED_FORMAT2(AssertQStringsEqual, instrumentPointing[1], "$mro/kernels/ck/mro_crm_psp_110223_101128.bc");
   EXPECT_PRED_FORMAT2(AssertQStringsEqual, instrumentPointing[2], "$mro/kernels/ck/mro_sc_psp_110222_110228.bc");
-  EXPECT_PRED_FORMAT2(AssertQStringsEqual, instrumentPointing[3], "$mro/kernels/fk/mro_v15.tf");
+  // Use a regex to match the versin # for the frame kernel because this sometimes updates
```
Comment (acpaquette) [inline isis/tests/FunctionalTestsSpiceinit.cpp]: Fix small typo here, versin to version.

**Review by acpaquette — DISMISSED** [review]
Small thing to fix but overall looks good!

**Review by acpaquette — APPROVED** [review]

**Comment by jessemapel** [comment]
kerneldbgen should be passing now too

**Pull request merged**